how-to/customize-workspace/scripts/webpack-parallel.mjs

```diff
@@ -0,0 +1,22 @@
+import { spawn } from 'child_process';
+import path from 'path';
+
+async function run(packageDir, configFile) {
+	console.log('packagerDir:', packageDir);
```

Comment:
Would this be impacted by order or would customize always be built first? if there isn't an order issue should this be the default rather than having another build command?

Reply:
order doesn't matter, as modules import the source directly, they don't need a built version. this is now the default, old mode is build-client-serial

Reply:
do we still need old mode?